qa-data/invalid-data-delimiter/data.csv

```diff
@@ -0,0 +1,3 @@
+id,city,,,,,
```

Comment:
I think this is not `invalid-data-delimiter`. In my understanding, invalid delimiter is, for instance, when you have `;` instead of default `,`